common/http_client.go

```diff
@@ -45,7 +45,7 @@ func GetBodyBytesFromResponse(response *http.Response) []byte {
 
 	statusCodeStartsWith2 := regexp.MustCompile(`2\d{2}`)
```

Comment:
Does is make sense to just compare the int status `200 <= response.StatusCode < 300`?